.license_check.yaml

```diff
@@ -0,0 +1,6 @@
+add_exclude:
+  - ".license_check.yaml"
+  - ".github/*"
+  - "Makefile"
```

Comment:
Does the Makefile need to be ignored?

Reply:
Need to... not strictly. but im getting awful annoyed at the license checker.  And it is JUST configuration.